.github/workflows/query-list.yml

```diff
@@ -0,0 +1,46 @@
+name: Build code scanning query list
+
+on:
+  push:
+    branches:
+     - master
+     - 'rc/**'
+     - querylist-python
```

Comment:
For future testing purposes, I suggest you add a
```
pull_request:
  paths:
    - '.github/workflows/query-list.yml'
```

Reply:
Done! I've also included the path to the script itself.